inception/inception-api-annotation-experimental/src/main/ts/client/AnnotationExperienceAPIImpl.ts

```diff
@@ -16,293 +16,448 @@
  * limitations under the License.
  */
 import {Client, Stomp} from '@stomp/stompjs';
-import {AnnotationExperienceAPIVisualization} from "./visualization/AnnotationExperienceAPIVisualization";
-import {AnnotationExperienceAPIActionHandler} from "./actionhandling/AnnotationExperienceAPIActionHandler";
-import {ServerMessage} from "./util/ServerMessage";
 import {AnnotationExperienceAPI} from "./AnnotationExperienceAPI"
-import {Annotation} from "./util/Annotation";
+import {Span} from "./model/Span";
+import {NewDocumentResponse} from "./messages/response/NewDocumentResponse";
+import {NewViewportResponse} from "./messages/response/NewViewportResponse";
+import {ErrorMessage} from "./messages/response/ErrorMessage";
+import {NewDocumentRequest} from "./messages/request/NewDocumentRequest";
+import {NewViewportRequest} from "./messages/request/NewViewportRequest";
+import {SelectSpanRequest} from "./messages/request/span/SelectSpanRequest";
+import {UpdateSpanRequest} from "./messages/request/span/UpdateSpanRequest";
+import {CreateSpanRequest} from "./messages/request/span/CreateSpanRequest";
+import {DeleteSpanRequest} from "./messages/request/span/DeleteSpanRequest";
+import {SelectSpanResponse} from "./messages/response/span/SelectSpanResponse";
+import {UpdateSpanResponse} from "./messages/response/span/UpdateSpanResponse";
+import {CreateSpanResponse} from "./messages/response/span/CreateSpanResponse";
+import {DeleteSpanResponse} from "./messages/response/span/DeleteSpanResponse";
+import {Relation} from "./model/Relation";
+import {UpdateRelationResponse} from "./messages/response/relation/UpdateRelationResponse";
+import {CreateRelationResponse} from "./messages/response/relation/CreateRelationResponse";
+import {SelectRelationResponse} from "./messages/response/relation/SelectRelationResponse";
+import {SelectRelationRequest} from "./messages/request/relation/SelectRelationRequest";
+import {DeleteRelationRequest} from "./messages/request/relation/DeleteRelationRequest";
+import {UpdateRelationRequest} from "./messages/request/relation/UpdateRelationRequest";
+import {CreateRelationRequest} from "./messages/request/relation/CreateRelationRequest";
+import {DeleteRelationResponse} from "./messages/response/relation/DeleteRelationResponse";
+import {AllSpanResponse} from "./messages/response/span/AllSpanResponse";
+import {AllRelationResponse} from "./messages/response/relation/AllRelationResponse";
+import {AllSpanRequest} from "./messages/request/span/AllSpanRequest";
+import {AllRelationRequest} from "./messages/request/relation/AllRelationRequest";
+import {Viewport} from "./model/Viewport";
 
 export class AnnotationExperienceAPIImpl implements AnnotationExperienceAPI {
 
     //Websocket and stomp broker
-    stompClient: Client;
-    connected: boolean = false;
+    private _stompClient: Client;
```

Comment:
Why the underscore?

Reply:
It depends on how we want to work in typescript: 
When we want to use a getters and setters then the "_" is necessary, because otherwise e.g.
get text(): string[] {
return this.text;
}
would call itself an infinite amount of times, instead of the property "text" (I've checked that).
However, we can also omit getters and setters and simply call in other classes:
experimentalAnnotationAPI.text
to obtain the value. In that case the underscore will not be needed. 
So yes sure, I can remove the getters and the underscores, no big deal.